test/fakeintake/server/server.go

```diff
@@ -300,16 +308,44 @@ func (fi *Server) handleGetPayloads(w http.ResponseWriter, req *http.Request) {
 		})
 		return
 	}
+
 	// we could support multiple endpoints in the future
 	route := routes[0]
-	log.Printf("Handling GetPayload request for %s payloads", route)
-	payloads := fi.safeGetPayloads(route)
-
-	// build response
-	resp := api.APIFakeIntakePayloadsGETResponse{
-		Payloads: payloads,
+	log.Printf("Handling GetPayload request for %s payloads.", route)
+	var jsonResp []byte
+	var err error
+	if req.URL.Query().Get("format") == "" {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should we be stricter here, by getting the format query option and defaulting to `raw` ? Currently asking for `payloads?format="raw"` would return `json` parsed data